README.md

```diff
@@ -15,3 +15,5 @@ part of the array each recursive call considers.
 Analyse the time complexity of your implementation and give a $\Theta$ bound for
 its worst-case runtime. Add your answer, including your reasoning, to this
 markdown file.
+
+If n is the size of the array that is to be sorted, the time complexity is equal to $\Theta (n log n)$. The nested for-loops both iterate $logn$ times because the size doubles through each iteration. In the inner-loop, the mergeArr function is called which merges the arrays and takes $\Theta (n)$ time. These together make the worst-case runtime equal to $\Theta (n log n)$.
```

Comment:
...so the same as the normal mergesort?

Reply:
Yes, they both have the same runtimes because they both have similar dividing and merging steps. In the iterative version, the array size is doubled at each iteration, which takes log(n) time. In the recursive version, the array is divided in half, which also takes log(n) time. They both have a time of n for merging.

Reply:
Ah, I see. Your version is not in place.

Reply:
So would the complexity change for the in-place version?